### PR TITLE
iOS TableView Fixes

### PR DESCRIPTION
**Pull request by adly-holler**
Cleans up the code, fixes scrollview callback, fixes selection diffing, more compliant with Apple

### CODE REVIEW

File: Few-iOS/ScrollView.swift [hunk 1]
```diff
@@ -23,9 +23,8 @@ private class FewScrollView: UIScrollView, UIScrollViewDelegate {
         fatalError("FewScrollView should not be used in a xib.")
     }
     
-    private func scrollViewDidScroll(scrollView: UIScrollView) {
-        let visibleRect = CGRect(x: contentOffset.x, y: contentOffset.y, width: contentOffset.x + bounds.size.width, height: contentOffset.y + bounds.size.height)
-        didScroll(visibleRect)
+    @objc func scrollViewDidScroll(scrollView: UIScrollView) {
+        didScroll(bounds)
```
Comment (joshaber) [inline Few-iOS/ScrollView.swift]: Ha, good call :+1:

File: Few-iOS/TableView.swift [hunk 2]
```diff
@@ -104,16 +101,21 @@ public class TableView: Element {
...
+            let handler = tableView.handler
             
             layoutElements()
             
             handler?.elements = elements
-            
-            let tableView = scrollView.handler?.tableView
-            
             handler?.selectionChanged = selectionChanged
+            if tableView.indexPathForSelectedRow()?.row != selectedRow {
```
Comment (joshaber) [inline Few-iOS/TableView.swift]: Since we end up calling `tableView.indexPathForSelectedRow()` twice, could we pull it out into a `let`?

**Comment by joshaber** [comment]
Fantastic. Just a single :memo:.

**Comment by adly-holler** [comment]
Sweet! I think I've got it dialed in

**Comment by joshaber** [comment]
:boom: Just needs a merge in from `master`.

**Comment by adly-holler** [comment]
Sir, all sectors are 🌵 for merge.

**Comment by robrix** [comment]
:star:

**Pull request merged**